react-pr2/src/features/Cart/components/AddressForm/index.jsx

```diff
@@ -0,0 +1,73 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Box, Button } from '@mui/material';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import * as yup from 'yup';
+import { useTranslation } from 'react-i18next';
+import InputField from '../..//../../components/form-controls/InputField';
+
+function AddressForm(props) {
+  const { t } = useTranslation();
+
+  const { onSubmit, activeStep, steps, handleBack } = props;
+  const schema = yup
+    .object()
+    .shape({
+      fullname: yup
+        .string()
+        .required('Please enter fullname')
+        .min(2, 'Mininum 2 characters')
+        .max(15, 'Maximum 15 characters'),
+      sdt: yup
+        .string()
+        .required('Please enter number phone')
+        .min(10, 'Please enter valid phone number')
+        .max(10, 'Please enter valid phone number')
+        .typeError('Please enter number phone'),
+      district: yup.string().required('Please enter district').min(3, 'district is too short!'),
+      ward: yup.string().required('Please enter ward/commune').min(2, 'ward is too short!'),
+      address: yup.string().required('Please enter address').min(3, 'address is too short!'),
+    })
+    .required();
+
+  const form = useForm({
+    defaultValues: {
+      fullname: '',
+      sdt: '',
+      district: '',
+      ward: '',
+      address: '',
+    },
+    resolver: yupResolver(schema),
+  });
+  const handleSubmit = (values) => {
+    onSubmit && onSubmit(values);
+  };
+
+  const handleBackCheckout = () => {
+    handleBack();
+  };
+
+  return (
+    <form onSubmit={form.handleSubmit(handleSubmit)}>
+      {t('Shipping address')}
+      <InputField name="fullname" label={`${t('Fullname')}`} form={form}></InputField>
+      <InputField name="sdt" label="SĐT" form={form}></InputField>
```

Comment:
```suggestion
      <InputField name="phone" label="SĐT" form={form}></InputField>
```
label i18n luôn chứ, ai lại để tiếng Anh là SĐT bao giờ :v